### PR TITLE
Fix PDF generation on windows

### PR DESCRIPTION
**Pull request by CAMOBAP795**

### CODE REVIEW

**Comment by ronaldtse** [comment]
Thanks @CAMOBAP795 ! Can you also help add an AppVeyor test to test it on Windows for this?

**Comment by CAMOBAP795** [comment]
> Can you also help add an AppVeyor test to test it on Windows for this?

Yep, on it

File: lib/metanorma/output/pdf.rb [hunk 1]
```diff
@@ -7,7 +8,13 @@ class Pdf < Base
       def convert(url_path, output_path)
         file_url = "file://#{Dir.pwd}/#{url_path}"
         pdfjs = File.join(File.dirname(__FILE__), '../../../bin/metanorma-pdf.js')
-        system "export NODE_PATH=$(npm root --quiet -g); node #{pdfjs} #{file_url} #{output_path}"
+        node_path = `npm root --quiet -g`.strip
+        if OS.windows?
+          ENV['NODE_PATH'] = node_path # for some reason 'set NODE_PATH=... && node' doesn't work
```
[the remark this answers is not in the document]
Comment (ronaldtse) [reply lib/metanorma/output/pdf.rb]: Ah! You're right, the correct way to set ENV on Ruby is via `ENV['xxx']`. I think regardless of platform, we should use `ENV['xxx']`. Can you help update it? Thanks!

**Comment by CAMOBAP795** [comment]
@ronaldtse you are absolutely right, such easy idea, IDK why I skipped it, on the way

File: lib/metanorma/output/pdf.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (CAMOBAP795) [reply lib/metanorma/output/pdf.rb]: Done

**Pull request merged**